asabiris/formatter/attachments.py

```diff
@@ -103,7 +103,16 @@ async def render_attachment(self, attachments, prefered_format='html'):
 					)
 
 				else:
-					raise FormatError(format=fmt)
+					raise ASABIrisError(
+						ErrorCode.INVALID_PATH,
+						tech_message="Incorrect template path '{}'. Move templates to '/Templates/Email/".format(
+							template),
+						error_i18n_key="Incorrect template path '{}'. Please move your templates to '/Templates/Email/".format(
```

Comment:
Hi watch out for double parenthesis in error_i18n_key: '{{path}}'... everywhere :)

Reply:
@language-shprt This is how the response looks

```
{
    "result": "ERROR",
    "error": "IrisError|Incorrect template path '{{incorrect_path}}'. Please move your templates to '/Templates/Email/",
    "error_dict": {
        "incorrect_path": "/Templates/Email/message.mp3"
    },
    "tech_err": "Incorrect template path '/Templates/Email/message.mp3'. Move templates to '/Templates/Email/"
}
```